runtime/oti/j9nonbuilder.h

```diff
@@ -252,6 +252,10 @@
 #define J9DescriptionCpTypeShift 0x4
 #define J9DescriptionImmediate 0x1
 
+/* Rom CP Flag for constant dynamic with primitive type */
+#define J9DescriptionCpPrimitiveType 1 << 20
```

Comment:
Please add brackets to this.  Also, why 20?

Reply:
the slot is formatted as
12 bit unused | 16 bit bsmIndex | 4 bit Cp descriptor
20 is the least significant bit unused

Reply:
Good info to put into the code